easybuild/easyconfigs/p/pymatgen/pymatgen-3.0.13-intel-2015a-Python-2.7.9.eb

```diff
@@ -0,0 +1,35 @@
+easyblock = "PythonPackage"
+
+name = 'pymatgen'
+version = '3.0.13'
+
+homepage = 'http://pypi.python.org/pypi/pymatgen'
+description = """Python Materials Genomics is a robust materials analysis code that defines core object
+ representations for structures and molecules with support for many electronic structure codes."""
+
+toolchain = {'name': 'intel', 'version': '2015a'}
+
+source_urls = [PYPI_SOURCE]
+sources = [SOURCE_TAR_GZ]
+
+python = 'Python'
+pythonversion = '2.7.9'
+
+versionsuffix = '-%s-%s' % (python, pythonversion)
+
+dependencies = [
+    (python, pythonversion),
+    ('spglib', '1.7.3'),
+]
+
+py_short_ver = '.'.join(pythonversion.split('.')[0:2])
+pylibdir = "lib/python%s/site-packages/%s" % (py_short_ver, name)
+
+sanity_check_paths = {
+    'files': ['bin/pmg', 'bin/ipmg',
+              ("%s-%%(version)s-py%s.egg" % (pylibdir, py_short_ver),
```

Comment:
it depends on the particular Python package being installed...

a template for this would be nice though (but sometimes there's an extra suffix, like `...-linux-x86_64.egg`)